setup.py

```diff
@@ -17,7 +17,7 @@ def read(*paths):
 
 
 PACKAGE = 'data_quality'
-INSTALL_REQUIRES = ['click>=6.2', 'goodtables>=0.6.5', 'pytz']
+INSTALL_REQUIRES = ['click>=6.2', 'goodtables>=0.6.5', 'pytz', 'datapackage']
```

Comment:
you are directly using the `jsontableschema` API, so you need to add it as a dependency too.